### PR TITLE
fix: Added missing dependencies for Modal, FilterDrawer, and Tile

### PR DESCRIPTION
**Pull request by ActuallyACat**
Does what it says on the box. To be released before https://github.com/cultureamp/kaizen-design-system/pull/772

### CODE REVIEW

File: draft-packages/filter-drawer/KaizenDraft/FilterDrawer/FilterDrawer.tsx [hunk 1]
```diff
@@ -2,6 +2,12 @@ import { MenuContent } from "@kaizen/draft-menu"
 import { StatelessMenu } from "@kaizen/draft-menu/KaizenDraft/Menu/StatelessMenu"
 import classnames from "classnames"
 import * as React from "react"
+/**
+ * Eslint throws a false positive for modules that use require. Ensure you
```
Comment (mbylstra) [inline draft-packages/filter-drawer/KaizenDraft/FilterDrawer/FilterDrawer.tsx:6]: I'm a bit confused by the comment - should it be "Eslint throws a false **negative**"? As in, the rule thows an error, but it shouldn't?

"before turning this rule off.." Isn't that what `// eslint-disable-next-line` does - it's already turned off?

Comment (mbylstra) [reply draft-packages/filter-drawer/KaizenDraft/FilterDrawer/FilterDrawer.tsx:6]: I think I might just be confused by the semantics of positive and negative 😆

Comment (mbylstra) [reply draft-packages/filter-drawer/KaizenDraft/FilterDrawer/FilterDrawer.tsx:6]: positive as in in good, or positive as in this is "positively bad" or "this positively fails the check"?

Comment (mbylstra) [reply draft-packages/filter-drawer/KaizenDraft/FilterDrawer/FilterDrawer.tsx:6]: maybe it could read "throws a false error" (if that's what it means)

Comment (ActuallyACat) [reply draft-packages/filter-drawer/KaizenDraft/FilterDrawer/FilterDrawer.tsx]: D'oh, you're right. It should read `false negative`, as in, there is an error reported when there shouldn't be. Good catch, i've now fixed this up.

**Review by mbylstra — APPROVED** [review]
Sweet, I think I get it now.

- It was throwing warnings but us developers were ignoring them
- upgrading to error would break the build because of the require false negatives
- so we need to ignore the errors/warnings for the require statements
- after all that, we still need to add react-dom and the component library
- now all is good!

**Pull request merged**